thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@ember/component';
+import moment from 'moment';
+
+export default Component.extend({
+
+  //
+  // external
+  //
+  onContext: null, // func (context)
+
+  onChart: null, // func (timeseriesMode)
+
+  context: null, // {}
+
+  timeseriesMode: null, // ""
+
+  //
+  // external (optional)
+  //
+  analysisRangeMax: moment().endOf('day'),
+
+  analysisRangePredefined: {
+    'Today': [moment(), moment().endOf('day')],
+    'Last 3 days': [moment().subtract(2, 'days').startOf('day'), moment().endOf('day')],
+    'Last 7 days': [moment().subtract(6, 'days').startOf('day'), moment().endOf('day')],
+    'Last 14 days': [moment().subtract(13, 'days').startOf('day'), moment().endOf('day')],
+    'Last 28 days': [moment().subtract(27, 'days').startOf('day'), moment().endOf('day')]
+  },
+
+  granularityOptions: ['5_MINUTES', '15_MINUTES', '1_HOURS', '3_HOURS', '1_DAYS'],
+
+  compareModeOptions: ['WoW', 'Wo2W', 'Wo3W', 'Wo4W'],
+
+  timeseriesModeOptions: ['absolute', 'relative', 'log', 'split'],
+
+  //
+  // internal
+  //
+  analysisRangeStart: null, // 0
+
+  analysisRangeEnd: null, // 0
+
+  granularity: null, // ""
+
+  compareMode: null, // ""
+
+  didReceiveAttrs() {
+    const { context } = this.getProperties('context');
```

Comment:
you can use `this.get` since you are only getting 1 property.
`const context = this.get('context')`